addons/web/static/tests/views/fields/properties_field_tests.js

```diff
@@ -1387,6 +1387,44 @@ QUnit.module("Fields", (hooks) => {
         }
     });
 
+    /**
+     * check if property field popover closes when clicking on delete property icon.
+     */
+    QUnit.test(
+        "properties: close property popover once clicked on delete icon",
+        async function (assert) {
+            async function mockRPC(route, { method, model, kwargs }) {
+                if (["check_access_rights", "check_access_rule"].includes(method)) {
+                    return true;
+                }
+            }
+            await makeView({
+                type: "form",
+                resModel: "partner",
+                resId: 3,
+                serverData,
+                arch: `
+                <form>
+                    <sheet>
+                        <group>
+                            <field name="company_id"/>
+                            <field name="display_name"/>
+                            <field name="properties" widget="properties"/>
+                        </group>
+                    </sheet>
+                </form>`,
+                mockRPC,
+            });
+
+            // We open the property popover
+            await click(target, ".o_property_field:first-child .o_field_property_open_popover");
+            assert.containsOnce(target,".o_field_property_definition");
+
+            //we click on delete button
```

Comment:
Picky: Can you do "// We" instead of "//we" for consistency with previous comment?